### PR TITLE
:memo:[#1626] Add JWT examples to docs

### LINKED ISSUE #1626: Document the usage of JWT tokens better, in various programming languages
### Thema / Theme

Other

### Omschrijving / Description

Discussed with Jan from Den Haag, lets improve the OpenZaak documentation regarding the usage of JWT's. Having a few examples of generating a JWT in various languages (PHP, Javascript etc) would help in API usage

### Toegevoegde waarde / Added value

_No response_

### Aanvullende opmerkingen / Additional context

_No response_

### PR DESCRIPTION
**Pull request by Coperh**
Fixes #1626

Added some links to JWT stuff.

Added examples for Python, PHP. Java and JavaScript

### CODE REVIEW

**Comment by Coperh** [comment]
@alextreme I've added examples for Python, PHP, Java, and JavaScript in https://github.com/open-zaak/open-zaak/blob/d4d6d6e473aa3e0532f21237394760d253304e1c/docs/client-development/authentication.rst

**Comment by Coperh** [comment]
Should the zaak example actually work? As in it should retrieve a zaaktype from test.openzaak.

File: docs/client-development/authentication.rst [hunk 1]
```diff
@@ -78,3 +78,169 @@ This would then be used in an API call like:
...
+                "iss": CLIENT_ID,
+                "iat": int(time.time()),  # current time in seconds
+                "client_id": CLIENT_ID,
+                "user_id": "eample@example.com",
+                "user_representation": "Example Name",
+            }
+            jwt_token = jwt.encode(payload, SECRET, algorithm="HS256")
+
+            # add token token to the authentication HTTP header of your request library
+            zaaktype_url = "https://test.openzaak.nl/catalogi/api/v1/zaaktypen/4acb5ab8-f189-4559-b18a-8a54553a74ff"
```
Comment (annashamray) [inline docs/client-development/authentication.rst:113]: `test.openzaak.nl` is a domain of Maykin test deployment for internal purporses, please change it to something like: `openzaak.gemeente.local` or `openzaak.example.org`

File: src/openzaak/utils/apidoc.py [hunk 2]
```diff
@@ -34,4 +34,11 @@
 `user_id` en `user_representation` zijn nodig voor de audit trails. Het zijn
 vrije velden met als enige beperking dat de lengte maximaal de lengte van
 de overeenkomstige velden in de audit trail resources is (zie rest API spec).
+
+**Handige JWT links**
```
Comment (annashamray) [inline src/openzaak/utils/apidoc.py:38]: I'dd suggest to leave here only a link to Authentication page at OZ RTD, and add other useful links there. Such way you will need to make changes only in RTD documentation in the future

**Review by annashamray — CHANGES_REQUESTED** [review]
I can't assess java and php, the rest looks good, just some minor remarks

File: docs/client-development/authentication.rst (hunk 1 above)
Comment (Coperh) [reply docs/client-development/authentication.rst]: Done

File: src/openzaak/utils/apidoc.py (hunk 2 above)
Comment (Coperh) [reply src/openzaak/utils/apidoc.py]: Just made it a single link at the bottom in possibly correct Dutch.

**Review by annashamray — APPROVED** [review]
:+1:

**Pull request merged**